jest.config.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
     '<rootDir>/test-env.js'
   ],
   testPathIgnorePatterns: [
-    'mocks.js'
+    'mocks.js',
+    'test_data.js'
```

Comment:
We had previously chosen `mocks.js` for this purpose, the data defined in `static/src/js/components/GranuleDetails/__tests__/test_data.js` is defined exactly like the data in `/static/src/js/util/request/__tests__/mocks.js`. I don't care which we choose, but we should stick to an expected filename for mocked data -- we should should remove one of these and rename the files appropriately.

Reply:
I recall a conversation where we had talked about how this sort of data was different than what had been set in the mocks.js file. Thats the only reason I had set it this way. I may have misunderstood. Totally fine with using _mocks.js_.